File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_Two/C++/prog2.cpp
#include<iostream>

int main() {
	int num1, num2, fact;
	std::cout << "Enter num1 : ";
	std::cin >> num1;
	std::cout <<"Enter num2 : ";
	std::cin >> num2;
	for(int i = num1; i <= num2; i++) {
		fact = 1;
		for(int j = 1; j <= i; j++) {
			fact *= j;
		}
		std::cout <<"Fact of " << i << " = " << fact << "\n";
	}
	return 0;
}

File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_Two/Python/prog2.py
num1, num2 = [int(i) for i in input("Input : ").split()]
for i in range(num1, num2 + 1):
    fact = 1
    for j in range(1, i + 1):
        fact *= j 
    print("Fact of ", i, " = ", fact)
